src/dump.js

```diff
@@ -60,10 +60,10 @@ export default ( function() {
 			parse: function( obj, objType, stack ) {
 				stack = stack || [];
 				var res, parser, parserType,
-					inStack = inArray( obj, stack );
+					objIndex = stack.indexOf( obj );
 
-				if ( inStack !== -1 ) {
-					return "recursion(" + ( inStack - stack.length ) + ")";
+				if ( objIndex !== -1 ) {
+					return "recursion(" + ( objIndex - stack.length ) + ")";
```

Comment:
This would probably work better as a template literal, since you're using them elsewhere in this PR.